lib/acmesmith/command.rb

```diff
@@ -64,6 +64,13 @@ def request(common_name, *sans)
 
       puts cert.certificate.to_text
       puts cert.certificate.to_pem
+
+      execute_post_issue_hooks(common_name)
+    end
+
+    desc "post_issue_hooks COMMON_NAME", "Run all post-issueing hooks for common name. (for testing purposed)"
```

Comment:
- Use `-` instead of `_` for command names
- I prefer name `run-post-issue-hooks`